R/modal.R

```diff
@@ -0,0 +1,58 @@
+#' Create Semantic UI modal
+#'
+#' This creates a modal using Semantic UI styles.
+#'
+#' @param id. ID to be added to the modal div. Default "".
+#' @param class. Classes except "ui modal" to be added to the modal. Semantic UI classes can be used. Default "".
+#' @param header. Content to be displayed in the modal header. Default "".
+#' @param content. Content to be displayed in the modal body. Default "".
+#' @param footer. Content to be displayed in the modal footer. Usually for buttons. Default NULL.
+#' @param target. Javascript selector for the element that will open the modal. Default NULL.
+#' @param settings. List of vectors of Semantic UI settings to be added to the modal. Default NULL.
```

Comment:
why there are dots after parameter names? This is why CI is failing

Reply:
Thank you that one was bugging me out :)